### PR TITLE
Add build configurations

### PR DESCRIPTION
**Pull request by cmgutmanis**

### CODE REVIEW

**Comment by cmgutmanis** [comment]
@cmgutmanis

**Comment by cmgutmanis** [comment]
@alexsab1 @jrodsmitty @KristyMensch @SirkleZero

File: Sitecore.Courier.Runner/ExclusionHandler.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+        // validates that an exclusion file is specified and exists if a build configuration is specified in argument parameters.
+        public static bool HasValidExclusions(string buildConfiguration, string xmlFilePath)
+        {
+            if (string.IsNullOrEmpty(buildConfiguration))
+            {
+                return false;
+            }
+
+            if (!string.IsNullOrEmpty(buildConfiguration) && string.IsNullOrEmpty(xmlFilePath))
```
Comment (jrodsmitty) [inline Sitecore.Courier.Runner/ExclusionHandler.cs]: @cmgutmanis The first condition isn't needed here, per line 55. You'd never get to this line if `buildConfiguration` is empty or null, so I believe it's redundant.

File: Sitecore.Courier.Runner/Program.cs [hunk 2]
```diff
@@ -1,8 +1,11 @@
-﻿namespace Sitecore.Courier.Runner
+﻿using Sitecore.Shell.Applications.ContentEditor;
```
Comment (jrodsmitty) [inline Sitecore.Courier.Runner/Program.cs]: Consider moving this inside the namespace for consistency. Or vise-versa

File: Sitecore.Courier.Runner/ExclusionHandler.cs (hunk 1 above)
Comment (cmgutmanis) [inline Sitecore.Courier.Runner/ExclusionHandler.cs]: nice catch, thanks

**Pull request merged**